src/main/java/org/jabref/gui/entryeditor/SourceTab.java

```diff
@@ -233,7 +217,12 @@ private void updateCodeArea() {
 
             codeArea.clear();
             try {
-                codeArea.appendText(getSourceString(currentEntry, mode, fieldPreferences));
+                String sourceText = getSourceString(currentEntry, mode, fieldPreferences);
+                LOGGER.info("sourceText length: {}", sourceText.length());
```

Comment:
make this debug, info is too much noise

Reply:
I will change it, it was only for debugging